garage/envs/wrappers/fire_reset.py

```diff
@@ -0,0 +1,28 @@
+"""Fire reset wrapper for gym.Env."""
+import gym
+
+
+class FireReset(gym.Wrapper):
```

Comment:
again, this wrapper seems really environment-specific, which is no the intended purpose of wrappers.

It seems strange to me that you would even have a wrapper which takes a specific action on reset. Isn't that the role of the RL agent?